### ISSUE TITLE
 As a user, I want to upload a pair of files via a file browse-and-select interface

### ISSUE THREAD

**Issue by theecrit** [openoakland/woeip#129]
*Description*
I have a CSV and log file from a single session, and I’d like to upload them to the server as seamlessly as possible using a file browse-and-select model per the attached happy path prototype (below).

**Note:** this issue is a counterpart to #184. Whichever is complete first should serve as the basis for the other.

*Acceptance criteria* 
- [x] Upload interface includes affordance for file browse-and-select per happy path prototype below.
- [x] User can see a list of and remove selected files from the interface prior to confirming upload.
- [ ] All open questions in Issue have been answered or moved into new Issues.

*Dependencies and assumptions*
- ~User must be authenticated (signed in).~ We'll add the account gateway once the front-end is in place.
- System uploads files to a server-side directory (same server that's hosting the database).
- Status notifications and progress indicators will be covered by a separate issue.
- View will be built in React.
- User can only upload one pair at a time.

*Related documentation*
- [Prototype: happy path](https://www.figma.com/proto/IZw8VvX1CsRc764CTvg4aRSU/WOAQ-Wireframes-Parent-File?node-id=340%3A5671&scaling=contain)
- [Prototype: error path example](https://www.figma.com/proto/IZw8VvX1CsRc764CTvg4aRSU/WOAQ-Wireframes-Parent-File?node-id=341%3A5663&scaling=contain) for reference only; not covered by this issue (Note: See #131 for list of error use cases)
- [WOAQ-Data-Flow-Diagram-v5.pdf](https://github.com/openoakland/woeip/files/4157171/WOAQ-Data-Flow-Diagram-v5.pdf)
- [WOAQ-Upload-v5-Wireframes.pdf](https://github.com/openoakland/woeip/files/3937179/WOAQ-Upload-v5-Wireframes.pdf)
- ~[WOAQ-Upload-v4](https://drive.google.com/open?id=1Al5ILibeITvFcentrr0ia9R5QLRs9pIq)~

*Questions?*
- For back-end questions, reach out to @jayqi
- For visual design questions, reach out to Niels or Kimberly Low in #WOAQ Slack channel.
- For UX/flow, content, or anything else, reach out to @theecrit

**Comment by theecrit** [comment]
Update: currently waiting on upload API endpoint to wrap up: https://github.com/openoakland/woeip/issues/145

**Comment by theecrit** [comment]
Updated wireframes based on more recent development decisions captured here: https://github.com/openoakland/woeip/issues/132

**Comment by theecrit** [comment]
@ifed3 Also, does this issue need to be updated with any other changes/progress?

**Comment by theecrit** [comment]
Added data flow diagram per 12/8/19 developer discussion.

**Comment by theecrit** [comment]
@ifed3 Ask and ye shall receive. Prototype links have been added — let me know if you run into any other questions.

**Comment by joshling1919** [comment]
Using https://www.figma.com/file/YJGrbzkh0dei32uAH3Q5m3/WOAQ-Data-Flow?node-id=165%3A247 instead of the "Prototype: happy path" mentioned in the issue.

**Comment by theecrit** [comment]
Completed alongside #184.
